src/view/treeNodes/descriptionNode.ts

```diff
@@ -32,12 +32,27 @@ export class DescriptionNode extends TreeNode implements vscode.TreeItem {
 			arguments: [this],
 		};
 
-		this.contextValue = 'description';
+		this.registerSinceReviewChange();
+
+		this.updateContextValue();
 		this.tooltip = `Description of pull request #${pullRequestModel.number}`;
 		this.accessibilityInformation = { label: `Pull request page of pull request number ${pullRequestModel.number}`, role: 'button' };
 	}
 
 	getTreeItem(): vscode.TreeItem {
 		return this;
 	}
+
+	updateContextValue(): void {
+		this.contextValue = 'description' +
+			(this.pullRequestModel.hasChangesSinceLastReview ? ':changesSinceReview' : '') +
+			(this.pullRequestModel.showChangesSinceReview ? ':active' : ':inactive');
+	}
+
+	protected registerSinceReviewChange() {
+		this.pullRequestModel.onDidChangeChangesSinceReview(_ => {
```

Comment:
This event listener should be disposed so that we don't leak listeners. The `TreeNode` has a way to do this:

https://github.com/microsoft/vscode-pull-request-github/blob/f2ea88b0d3fab2e6c2c8a6952b628abca5eeaa83/src/view/treeNodes/fileChangeNode.ts#L120-L126